### PR TITLE
RHTAP-2994 Jenkins tests for GitHub

### PR DESCRIPTION
**Pull request by jkopriva**

### CODE REVIEW

**Comment by flacatus** [comment]
I really like the solution. We are setting up the new infra for rhtap e2e. Once is ready we can merge it!

Question:

Do we really need for every language?

**Comment by jkopriva** [comment]
@flacatus Thank you! Right now the Jenkins stuff is disabled. I have another commit for GitLab ready, just waiting for this one to be merged.
Ad languages: I am thinking, what if we in next PR(s) rework tests to have some smoke suite? You can run half of the languages for GitHub and the other half on GitLab? You will still need to run all tests for release testing, but CI on PRs could run only smoke and weekly job will also run full suite? WDYT?

**Review by prietyc123 — CHANGES_REQUESTED** [review]

File: src/apis/git-providers/github.ts
Comment (jkopriva) [reply src/apis/git-providers/github.ts:140]: Done, resolving

File: src/apis/ci/jenkins.ts
Comment (jkopriva) [reply src/apis/ci/jenkins.ts]: Fixed, resolving

File: jest.config.js
Comment (jkopriva) [reply jest.config.js]: Fixed, resolving!

**Comment by jkopriva** [comment]
@prietyc123 Thank you for review and testing!

**Review by flacatus — APPROVED** [review]

**Pull request merged**
